### ISSUE TITLE
Include webhook events https://github.com/kgajera/stripe-event-types

### ISSUE THREAD

**Issue by crobinson42** [stripe/stripe-node#1664]
### Is your feature request related to a problem? Please describe.

Webhook event TypeScript typings

### Describe the solution you'd like

This should be included in this repo: https://github.com/kgajera/stripe-event-types

### Describe alternatives you've considered

dying from manually assigning type annotations.

**Comment by anniel-stripe** [comment]
Hi, thanks for the feature request! We likely won't get around to this immediately, but this is a great idea.

**Comment by zerefdev** [comment]
> Hi, thanks for the feature request! We likely won't get around to this immediately, but this is a great idea.

Is there a reason not to get around this immediately/soon?
`event.data.object` typed as `{}` is annoying.

**Comment by anniel-stripe** [comment]
We certainly want this feature and hope to get to it soon! However, we’d like to generate webhook event types to ensure completeness / accuracy for users, but [the OpenAPI spec](https://github.com/stripe/openapi) which our generator uses as input does not define webhook events yet. This needs to happen first before we can generate webhook event type definitions, and unfortunately it’s not a trivial change.

**Comment by pakrym-stripe** [comment]
Closing as a duplicate of https://github.com/stripe/stripe-node/issues/902
